README.md

```diff
@@ -205,6 +143,7 @@ Have fun creating and using this plugin and the Nexus platform, we are glad to h
 
 Looking to contribute to our code but need some help? There's a few ways to get information:
 
+* If using Nexus Repository Manager 3.21+ or later please file any issues at https://issues.sonatype.org/.
 * Chat with us on [Gitter](https://gitter.im/sonatype/nexus-developers)
 * Check out the [Nexus3](http://stackoverflow.com/questions/tagged/nexus3) tag on Stack Overflow
-* Check out the [Nexus Repository User List](https://groups.google.com/a/glists.sonatype.com/forum/?hl=en#!forum/nexus-users)
+* Check out the [Nexus Repository User List](https://groups.google.com/a/glists.sonatype.com/forum/?hl=en#!forum/nexus-users)
```

Comment:
NOTE: I've heard we are pointing people to community.sonatype.com in favor of the google group.  @DarthHater can you confirm with @jlstephens89 out?
Non-blocker but might as well change if we're changing things anyway.